packages/core/src/auth/credential-provider.ts

```diff
@@ -169,11 +241,11 @@ export class EnvMomentoTokenProvider extends StringMomentoTokenProvider {
         `Missing required environment variable ${props.environmentVariableName}`
       );
     }
-    const decodedToken = decodeAuthToken(authToken);
     super({
-      authToken: decodedToken.authToken,
-      controlEndpoint: props.controlEndpoint ?? decodedToken.controlEndpoint,
-      cacheEndpoint: props.cacheEndpoint ?? decodedToken.cacheEndpoint,
+      authToken: authToken,
+      controlEndpoint: props.controlEndpoint,
+      cacheEndpoint: props.cacheEndpoint,
+      vectorEndpoint: props.vectorEndpoint,
```

Comment:
Hmm I am unable to find in this PR where this is actually set to `vector.blah...`

Reply:
https://github.com/momentohq/client-sdk-javascript/pull/786/files#diff-df5a3d98528b9dd05cb526fae3b1a3119deb52f12e5815a23c1867200e43f934R63